src/Microsoft.HttpRepl.Fakes/NullPreferences.cs

```diff
@@ -22,7 +22,12 @@ public int GetIntValue(string preference, int defaultValue = default)
             return defaultValue;
         }
 
-        public string GetValue(string preference, string defaultValue = default)
```

Comment:
We just pushed a change to this yesterday, changing the `string defaultValue = null` to `string defaultValue = default`. Guessing the PRs just crossed paths. Can you change that back here?

Reply:
sorry for missing that

Reply:
No worries. Thanks for the quick reply!